### ISSUE TITLE
pretrained model's performance on VOC

### ISSUE THREAD

**Issue by caiqi** [zhreshold/mxnet-ssd#121]
Hi, is the pretrained model in this repo trained with mexnet-ssd or converted from other implementation? I have tried training VOC 2007+2012 trainval using lr=0.004, batch_size=32 on a single gpu but the performance can't reach 77.*. It's about 73.*. Can you give some suggestions on reproducing the results of 77.*. Thanks.

**Comment by zhreshold** [comment]
which network, if it's vgg then yes, it is broken after last refactor

**Comment by caiqi** [comment]
Yes, it's vgg reduced 300. If the current version is broken, can I get the results with the repo before commit https://github.com/zhreshold/mxnet-ssd/commit/6f517aa1818c0ac37c1ddb8ed1858b6d9e49551a ? This commit seems change a lot of things.  Another question is that I compared this repo with this folder https://github.com/apache/incubator-mxnet/tree/master/example/ssd and there are some differences. Which one do you suggest to use? Thanks.

**Comment by BunnyShan** [comment]
hi, @zhreshold is resnet50 broken? I can't reproduce the result either. Could you plz give me some suggestion?
